server/routes/induction/create/reasonsNotToGetWorkCreateController.test.ts

```diff
@@ -0,0 +1,190 @@
+import { Request, Response } from 'express'
+import type { SessionData } from 'express-session'
+import type { InductionDto } from 'inductionDto'
+import type { ReasonsNotToGetWorkForm } from 'inductionForms'
+import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
+import ReasonsNotToGetWorkCreateController from './reasonsNotToGetWorkCreateController'
+import ReasonNotToGetWorkValue from '../../../enums/reasonNotToGetWorkValue'
+import HopingToGetWorkValue from '../../../enums/hopingToGetWorkValue'
+
+describe('reasonsNotToGetWorkCreateController', () => {
+  const controller = new ReasonsNotToGetWorkCreateController()
+
+  const prisonNumber = 'A1234BC'
+
+  let req: Request
+  let res: Response
+  const next = jest.fn()
+
+  const noErrors: Array<Record<string, string>> = []
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    req = {
+      path: `/prisoners/${prisonNumber}/create-induction/reasons-not-to-get-work`,
+      session: {} as SessionData,
+      body: {},
+      user: {} as Express.User,
+      params: { prisonNumber } as Record<string, string>,
+      flash: jest.fn(),
+    } as unknown as Request
+
+    res = {
+      redirect: jest.fn(),
+      render: jest.fn(),
+    } as unknown as Response
+  })
+
+  describe('getReasonsNotToGetWorkView', () => {
+    it('should get the reasons not to get work create view', async () => {
+      // Given
+      const prisonerSummary = aValidPrisonerSummary()
+      req.session.prisonerSummary = prisonerSummary
+
+      const inductionDto: InductionDto = {
+        prisonNumber,
+        workOnRelease: {
+          hopingToWork: HopingToGetWorkValue.NO,
+        },
+      } as InductionDto
+      req.session.inductionDto = inductionDto
+
+      req.session.reasonsNotToGetWorkForm = undefined
+
+      const expectedReasonsNotToGetWorkForm: ReasonsNotToGetWorkForm = {
+        reasonsNotToGetWork: [],
+      }
+
+      req.session.pageFlowHistory = {
```

Comment:
This test shouldnt need the page flow history any more - I think you can remove this from here